src/app.ts

```diff
@@ -143,7 +143,7 @@ async function app() {
 
   // Load dependencies
   const defs = new FHIRDefinitions();
-  const dependencyDefs = loadExternalDependencies(defs, config);
+  await loadExternalDependencies(defs, config);
```

Comment:
TBH, I don't remember for sure.  I think it probably was because I changed the signature to return `Promise<void>` and I couldn't think of what to call the variable in that case -- and then I realized we don't do anything with it but just `await` it a few lines down.  So I probably "streamlined" it by making 2 lines into one and eliminating the need to figure out the name of the variable.

I don't know how much of a performance difference it really makes in this case, but I can move the await back down below `loadCustomResources`.

Reply:
Yeah I don't think the performance difference would really be considerable, I was just wondering why the change. I leave it up to you whether you think the simplicity of the new code is worth the maybe slight performance difference.